src/UnityExtension/Assets/Editor/GitHub.Unity/UI/PUblishView.cs

```diff
@@ -91,6 +98,8 @@ private void PopulateView()
 
                             owners = owners.Union(organizationLogins).ToArray();
                         });
+                    }).Finally(task => {
+                        isBusy = false;
```

Comment:
Oh, this needs to be `FinallyInUI`, serialized ui data should only be touched in thread